src/Analyzers/CSharp/Tests/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -548,5 +548,38 @@ internal struct S1 { }
                 LanguageVersion = LanguageVersion.CSharp10,
             }.RunAsync();
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
+        [WorkItem(55703, "https://github.com/dotnet/roslyn/issues/55703")]
+        public async Task TestPartial_WithExistingModifier()
```

Comment:
do we need to do something similar for VB?

Reply:
This test was just a regression check to ensure the analyzer didn't have the same issue as the new document handler, and it doesn't. The new document handler doesn't have a VB implementation (and as Dustin pointed out in the issue, VB doesn't require the partial modifier in both places so it probably never should)